packages/core/test/tracing/spanHandle_test.js

```diff
@@ -0,0 +1,33 @@
+/*
+ * (c) Copyright IBM Corp. 2023
+ */
+
+'use strict';
+
+const { expect } = require('chai');
+const spanHandle = require('../../src/tracing/spanHandle');
+
+describe('tracing/spanHandle', () => {
+  const SpanHandle = spanHandle._SpanHandle;
+  const NoopSpanHandle = spanHandle._NoopSpanHandle;
+
+  it('NoOpSpanHandle should have the same methods as SpanHandle', () => {
+    verifyMethods(SpanHandle, NoopSpanHandle);
+  });
+
+  it('SpanHandle should have the same methods as NoOpSpanHandle', () => {
+    verifyMethods(NoopSpanHandle, SpanHandle);
+  });
+
```

Comment:
FYI: I plan to add a couple more unit tests for `annotate` and `findAndAnnotateErrorMessage` here as well later today (so please don't merge yet :-) )